topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -58,102 +58,179 @@ GWS facilitates analysis repeatability, while minimizing the number of manual st
 
 # Getting started on Galaxy
 
-This tutorial assumes you are comfortable getting data into Galaxy, running jobs, managing history, etc. If you are unfamiliar with Galaxy, we recommed you visit the [Galaxy Training Network](https://training.galaxyproject.org). Consider starting with the following trainings:
+--------
+
+This tutorial assumes you are comfortable getting data into Galaxy, running jobs, managing history, etc. If you are unfamiliar with Galaxy, we recommend you visit the [Galaxy Training Network](https://training.galaxyproject.org). Consider starting with the following trainings:
 - [Introduction to Galaxy]({% link topics/introduction/tutorials/introduction/slides.html %})
 - [Galaxy 101]({% link topics/introduction/tutorials/galaxy-intro-101/tutorial.md %})
 - [Getting Data into Galaxy]({% link topics/galaxy-interface/tutorials/get-data/slides.html %})
 - [Using Dataset Collections]({% link topics/galaxy-interface/tutorials/collections/tutorial.md %})
-- [Introduction to Galaxy Analyses](https://training.galaxyproject.org/training-material/topics/introduction)
```

Comment:
you can keep this! it just needed to be `{% link topics/introduction/index.md %}`